Carbon.WebApplication.SolutionService/Services/SolutionRegistrationService.cs

```diff
@@ -67,7 +67,7 @@ public async Task AddAsFeatureSet(FeatureSetCreationRequest data)
                 throw new IncompatibleSolutionException((int)ExceptionEnums.FeatureSetNameNull, "SolutionCreationRequest SolutionName cannot be Null or Default");
             }
 
-            var sendEp = await _busControl.GetSendEndpoint(new Uri("queue:featureset-creation-state"));
```

Comment:
Should we need to check is high avaible queue enabled?

Reply:
Quorum and the classic one works best with exchange: flag. Otherwise, quorum one throws exception.